templates/repo/issue/view_content.tmpl

```diff
@@ -55,20 +55,19 @@
 						</div>
 						<div id="comment-{{.Issue.ID}}" class="raw-content hide">{{.Issue.Content}}</div>
 						<div class="edit-content-zone hide" data-write="issue-{{.Issue.ID}}-write" data-preview="issue-{{.Issue.ID}}-preview" data-update-url="{{$.RepoLink}}/issues/{{.Issue.Index}}/content" data-context="{{.RepoLink}}" data-attachment-url="{{$.RepoLink}}/issues/{{.Issue.Index}}/attachments" data-view-attachment-url="{{$.RepoLink}}/issues/{{.Issue.Index}}/view-attachments"></div>
+						{{if .Issue.Attachments}}
+							<div class="ui clearing divider"></div>
+                            <div class="ui middle aligned padded grid">
+								{{template "repo/issue/view_content/attachments" Dict "ctx" $ "Attachments" .Issue.Attachments}}
+							</div>
```

Comment:
Missing a closing <div> here and the indenting is different from the rest of the file.

Reply:
Indentation should be fixed now. I can't see where there should be a missing `</div>`, though.